ansible/aix61-standalone/manualBootstrap.md

```diff
@@ -12,16 +12,114 @@ chfs -a size=+250000 /
 chfs -a size=917504 /tmp
 ```
 
-AIX72: unneeded: -1 and -2 and aix7-2 have different sizes, and 7.2 is big
-enough already:
+For the systems without ramdisks `/home` needs to be set to the same size as `/home/iojs/build` would be on a ramdisk machine
+
+```bash
+chfs -a size=23000000 /home
+```
+
+## Edit SSH config
+
+Add the following two lines to `/etc/ssh/sshd_config`:
+
+```
+PasswordAuthentication no
+ChallengeResponseAuthentication no
+```
+
+## Remove failedlogin file
+
+If /etc/security/failedlogin is growing without bounds on AIX6.1, remove
+it with a cron job, use `crontab -e`, and add one line:
+```sh
+0 12 * * * /usr/bin/rm -f /etc/security/failedlogin
+```
+
+## Fix "Missing" shared objects
+
+On the 7.1 machines we were facing this issues when yum installed packages were not able to find some of the shared objects they needed
+
+```sh
+bash-5.0$ gmake -v
+exec(): 0509-036 Cannot load program gmake because of the following errors:
+        0509-022 Cannot load module /opt/freeware/lib/libintl.a(libintl.so.8).
+        0509-150   Dependent module /usr/lib/libiconv.a(libiconv.so.2) could not be loaded.
+        0509-152   Member libiconv.so.2 is not found in archive
+        0509-022 Cannot load module make_64.
+        0509-150   Dependent module /opt/freeware/lib/libintl.a(libintl.so.8) could not be loaded.
+        0509-022 Cannot load module .
+```
+
+The fix is as following:
+
+```sh
+sudo rm /usr/lib/libiconv.a && sudo ln -s /opt/freeware/bin/libiconv.a /usr/lib
+```
```

Comment:
Is it possible to do this through ansible instead of a manual step?

Reply:
Its not safe to do, it could really damage the system. I'm not sure how badly, but possibly make ever executable unrunnable. I did it because the upstream yum packages don't seem to do the right thing here and had to figure out a way to make make build, and only after manual verification that the .so's (which are stored _inside_ static archives) were bitwise identical. It could be useful to add some info here: `ar x libiconv.a` on the two archives will extract  the .so from them. In this system (but not the aix7.2 I worked on 2 months ago, from the same yum package repo) the .a file in /opt was a pure superset of the one in /lib, so symlinking works.